README.md

```diff
@@ -15,7 +15,16 @@ git clone https://github.com/ddkang/aidb.git
 cd aidb
 pip install -r requirements.txt
 
+# We support PostgreSQL, MySQL, and SQLite
+# Change "sqlite" to "postgresql" or "mysql" if you'd like to use those
+pip install -r requirements/sql/requirements-sqlite.txt
```

Comment:
Isn't there a way to do something like "pip install aidb[sqlite, mysql]" or whatever?